docs/lms_config.ini

```diff
@@ -7,7 +7,7 @@ version = latest
 
 # below are the pdf settings for the pdf file
 [pdf_settings]
-pdf_base = https://media.readthedocs.io/pdf/open-edx-learner-guide
+pdf_base = https://media.readthedocs.org/pdf/open-edx-learner-guide
```

Comment:
Hmm.. I'm getting a 403 when accessing this site.
See my comment above about removing `pdf` altogether since it never got used.